Mathlib/Algebra/Module/Submodule/Pointwise.lean

```diff
@@ -546,4 +546,13 @@ lemma coe_span_smul {R' M' : Type*} [CommSemiring R'] [AddCommMonoid M'] [Module
 
 end set_acting_on_submodules
 
+lemma span_singleton_toAddSubgroup_eq_zmultiples (a : ℤ) :
+    (span ℤ {a}).toAddSubgroup = AddSubgroup.zmultiples a := by
```

Comment:
I don't understand why only the `Ideal.span` version is marked `@[simp]`. Shouldn't it be the other way round?

Reply:
I had both marked as `@[simp]`. CI complained that `simp` could prove the first one, so I removed the `@[simp]` marking on it. And since `simp` turns `Submodule.span` into `Ideal.span`, having only the first as a `simp` lemma probably wouldn't be a good idea (whether it applies would depend on what order `simp` lemmas are applied in).